### ISSUE TITLE
Can't login on Windows 10

### ISSUE THREAD

**Issue by LunarTwilight** [keybase/keybase-issues#2632]
<img width="960" alt="capture" src="https://cloud.githubusercontent.com/assets/13520392/19540275/a5d5809e-960a-11e6-99b7-d9d7210c4564.PNG">

**Comment by cjb** [comment]
CC @zanderz @taruti

**Comment by LunarTwilight** [comment]
The error went away once I restarted, so feel free to close this if you wish~

**Comment by zanderz** [comment]
I see that we use a homegrown function called `SafeWriteToDisk` which writes to a temp file and renames it to the target instead of just writing straight to the target. Maybe Windows doesn't like this if the target is still open? I opened a ticket to at least add a Go test for this method, but otherwise it's unclear how to diagnose without a local repro.
https://keybase.atlassian.net/browse/CORE-4012

**Comment by maxtaco** [comment]
@LunarTwilight did you potentially switch between administrator and regular user at any point in the past? One theory is the ownership of the configuration files changed.

**Comment by LunarTwilight** [comment]
@maxtaco No, the account I am using is admin and is the only account on the computer.

**Comment by maxtaco** [comment]
@zanderz ^ this might help you repro the issue

**Comment by zanderz** [comment]
Was the provisioning attempted directly after installing? Can you do a `keybase log send`? Thanks.

**Comment by LunarTwilight** [comment]
I don't know if it was.
<img width="960" alt="5caebd536b784fc7ac88ef150cfa546a" src="https://cloud.githubusercontent.com/assets/13520392/19710994/0059d6ea-9adf-11e6-92ca-f7049a59389d.png">

**Comment by zanderz** [comment]
@LunarTwilight One other question: are you running any antivirus software which may be scanning files written by other programs? I've read in a forum that this can cause MoveFileEx() access denied failures. Forums also say that retrying usually makes it work, so that's what we'll try.

**Comment by LunarTwilight** [comment]
I am using Kaspersky Internet Security.

**Comment by wiskirz** [comment]
I am also having trouble logging on using the Windows 10 client. I receive this error.
<img width="593" alt="keybasebug" src="https://cloud.githubusercontent.com/assets/9092935/20693238/a3209a48-b5a2-11e6-988d-64ac39560441.png">

**Comment by maxtaco** [comment]
Try `keybase db nuke` on the command line
